pkg/cmd/runs/list/list.go

```diff
@@ -24,24 +33,31 @@ func New(c *cli.Config) *cobra.Command {
 			airplane runs list --task <slug> -o json
 		`),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return run(cmd.Root().Context(), c, slug)
+			return run(cmd.Root().Context(), c, cfg)
 		},
 	}
 
-	cmd.Flags().StringVarP(&slug, "task", "t", "", "Filter runs by task slug")
+	cmd.Flags().StringVarP(&cfg.slug, "task", "t", "", "Filter runs by task slug")
+	cmd.Flags().IntVar(&cfg.limit, "limit", 100, "If >0, returns at most --limit items.")
+	cmd.Flags().Var(&cfg.since, "since", "Filter runs by the time they were created at")
```

Comment:
Maybe something like this for since (and similar `before the given time` for until)?

```suggestion
	cmd.Flags().Var(&cfg.since, "since", "Filter runs created since the given time")
```